src/Console/Installer.php

```diff
@@ -156,7 +156,17 @@ public static function setFolderPermissions($dir, $io)
             }
         };
 
-        $worldWritable = bindec('0000000111');
+        $worldWritable = 007;
+        if ($worldWritable === 007) {
```

Comment:
>  You edit the above permissions and the message won't render.

If users are forced to edit the file - they can also just delete the lines generating the notice.

Seems sub-optimal to me (and unnecessary), I don't understand what this has to do with deployment.